xmtp_mls/src/storage/encrypted_store/mod.rs

```diff
@@ -667,78 +664,4 @@ pub(crate) mod tests {
         }
         EncryptedMessageStore::remove_db_files(db_path)
     }
-
-    // get two connections
-    // start a transaction
-    // try to write with second connection
-    // write should fail & rollback
-    // first thread succeeds
-    // wasm does not have threads
-    #[cfg_attr(not(target_arch = "wasm32"), tokio::test)]
-    #[cfg(not(target_arch = "wasm32"))]
-    async fn test_transaction_rollback() {
```

Comment:
Had to delete this test since the barriers in this test were trying to force something that should be impossible now and was causing a deadlock.

Reply:
right, because the barriers would never progress since the transaction never finishes. So we would only deadlock IRL if we somehow had a transaction that never finishes, which shouldn't be possible w/o async